lib/cli/src/frameworks/lit/js/Button.stories.js

```diff
@@ -0,0 +1,51 @@
+import './Button';
+import { html } from 'lit';
```

Comment:
For consistency with the rest
```suggestion
import { html } from 'lit';

import './Button';
```